lib/crypto_markets/view/crypto_markets.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+class CryptoMarkets extends StatelessWidget {
+  const CryptoMarkets({
+    super.key,
+    required List<String> filteredMarkets,
+    required BoxConstraints boxConstraints,
+  })  : cryptoMarkets = filteredMarkets,
+        constraints = boxConstraints;
+
+  final List<String> cryptoMarkets;
+  final BoxConstraints constraints;
+
+  Widget cryptoDesktopMarkets() {
+    return SizedBox(
+      height: constraints.maxHeight * 0.8 - 120,
+      child: ListView.builder(
+        padding: const EdgeInsets.only(top: 10),
+        physics: const BouncingScrollPhysics(),
+        itemCount: 10,
+        itemBuilder: (context, index) {
+          /// CryptoDesktopMarketsCard()
+        },
+      ),
+    );
+  }
```

Comment:
[This has been pointed out so many times do not return the widget](https://dcm.dev/docs/rules/flutter/avoid-returning-widgets/#:~:text=Warns%20when%20a%20method%2C%20function,processor%20time%20for%20the%20operations.) . Please Stop doing that.

Reply:
oops my fault